src/index.js

```diff
@@ -88,6 +87,9 @@ function getIssuerName({ issuer }, didMap) {
 }
 
 function getLikelyImage(s) {
```

Comment:
how did I let a variable called `s` get through before?

Reply:
not sure, probably from dodeepsearch